src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,17 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private final UserService userService = new UserService();
+
+    public UserService getUserService() {
```

Comment:
Why do you need this get method?

Reply:
to get the variable, Intellij IDEA indicated that it is better to make UserService object private and to get it i created the get method